packages/cli-kit/src/ui.ts

```diff
@@ -48,6 +48,31 @@ interface BaseQuestion<TName extends string> {
   result?: (value: string) => string | boolean
 }
 
+export interface SimpleTaskOptions {
+  title: string
+  success?: string
+  task: () => Promise<void | string>
+}
+
+/**
+ * Performs a task with the title kept up to date and stdout available to the
+ * task while it runs (there is no re-writing stdout while the task runs).
+ */
+export const simpleTask = async ({title, success: _success, task}: SimpleTaskOptions) => {
+  let success
+  started(title, logUpdate)
```

Comment:
Is this the only place from where we are using `started`? In that case I'd scope the formatting logic to this `ui` module, either in this same function or in a different function. We can extract it in the near future if the reusing need emerges.